src/control/settings/Settings.cpp

```diff
@@ -1536,14 +1536,6 @@ void Settings::setPresentationHideElements(string elements) {
 
 auto Settings::getPdfPageCacheSize() const -> int { return this->pdfPageCacheSize; }
 
-void Settings::setPdfPageCacheSize(int size) {
-    if (this->pdfPageCacheSize == size) {
-        return;
-    }
-    this->pdfPageCacheSize = size;
-    save();
-}
-
```

Comment:
Is the pdfPageCacheSize obsolete? If yes, also remove the getter and the type. 
If not mark this function as [[maybe_unused]

Reply:
No, it's not obsolete. The getter is still used in the `XournalView` class constructor and the `SidebarPreviewBase` constructor. So I will mark the getter as [[maybe_unused]].